src/api/contributors/contributors.test.js

```diff
@@ -3,23 +3,25 @@ const request = require('supertest');
 const app = require('../../app');
 
 describe('GET /contributors', () => {
-  it('Should respond with a 200 status code', async () => {
+  it('Should respond with a 200 status code', async done => {
     const { status, type, body } = await request(app).get('/contributors');
     expect(status).toEqual(200);
     expect(type).toEqual('application/json');
     expect(body.length).toBeGreaterThanOrEqual(0);
```

Comment:
This will always be true

Reply:
I think it is worth testing for so as to make sure it stays true even after modifications to the current controller.

Reply:
I can't think of a scenario where this will ever be false. Body can be empty, or I can not be empty. Either way it will return true.

Reply:
Got it, I didn't realize which line you were commenting in. Making that change now.